components/chatlist.js

```diff
@@ -4,7 +4,12 @@ import SearchBarMessages from './searchBarMessages'
 import { useNavigation } from '@react-navigation/native';
 import Conversation from './conversation/conversation.js';
 
-function ChatList({ currentUser, data, isDarkTheme }) {
+import data from '../data/data'
+
+function ChatList({ currentUser, friendsList, isDarkTheme }) {
+
+  const [list] = useState(friendsList)
```

Comment:
What is the point of storing this in state if it does not have a corresponding function to manipulate it?

Reply:
If I don't do this and someone doesn't have any friends my looping function crashes. It's mainly a temp fix if user has no friends.